src/api/cpp/cvc5.h

```diff
@@ -3167,7 +3167,7 @@ class CVC5_EXPORT Solver
    * When constructing datatypes, unresolved sorts are replaced by the datatype
    * sort constructed for the datatype declaration it is associated with.
    *
-   * @note Create unresolved sorts with Solver::mkUninterpretedSort().
+   * @note Create unresolved sorts with Solver::mkUnresolvedSort().
```

Comment:
It seems the documentation for mkDatatypeSort in Solver.java is not in sync with C++ API since this note is missing in Solver.java